airflow/providers/trino/operators/trino.py

```diff
@@ -79,3 +81,18 @@ def execute(self, context: 'Context') -> None:
         self.hook.run(
             sql=self.sql, autocommit=self.autocommit, parameters=self.parameters, handler=self.handler
         )
+
+    def on_kill(self) -> None:
+        if self.hook is not None and isinstance(self.hook, TrinoHook):
+            query_id = "'" + self.hook.query_id + "'"
```

Comment:
Does the following work?

```suggestion
            query_id = f"'{self.hook.query_id}'"
```